examples/units_and_coordinates/stereo_loc_sunpy.py

```diff
@@ -0,0 +1,47 @@
+"""
+===================================
+Getting the location of STEREO-A using sunpy
+===================================
+
+How to get the position of planetary bodies im the solar system using
+`astropy's solar system ephemeris <http://docs.astropy.org/en/stable/coordinates/solarsystem.html#solar-system-ephemerides>`__ information and sunpy.
+"""
+import datetime
+
+import matplotlib.pyplot as plt
+
+from astropy.time import Time
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+today = datetime.datetime.now()
+obstime = Time(today)
+
+##############################################################################
+# Lets grab the positions of STEREO-A in Heliographic Stonyhurst
+# coordinates.
+
+
+aia = get_horizons_coord('STEREO-A', obstime)
+
+##############################################################################
+# Lets grab the positions of each of the planets in Heliographic Stonyhurst
+# coordinates.
+
+planet_list = ['mars', 'sun', 'earth']
+planet_coord = [get_body_heliographic_stonyhurst(
```

Comment:
You will want to explain why, one function was used for STEREO and one for the bodies.

Reply:
Because When I'm using the same function which is `get_horizons_coord()`  for finding the location of the bodies then I'm getting an Ambiguous target name Error which can be solved when we use unique IDs for bodies but then unique IDs are getting printed in the plot label. So that's why I used `get_horizons_coord()` for STEREO-A and `get_body_heliographic_stonyhurst()` for other bodies.
Please correct me if I'm using the wrong approach.

Reply:
I mean, you will want to explain this in the example.

Reply:
Ah, my bad.
Done!